src/NLog/LayoutRenderers/ThreadNameLayoutRenderer.cs

```diff
@@ -59,6 +59,6 @@ private static string GetStringValue()
 #endif
         }
 
-        string IStringValueRenderer.GetFormattedString(LogEventInfo _) => GetStringValue();
```

Comment:
`_` is fine here?

Reply:
I know. But it is reported as warning by Sonar: 

https://sonarcloud.io/project/issues?branch=dev&id=nlog&open=AWb1jYMNW3Y69bz5TTB3&resolved=false

Reply:
mmm disagree on that. Unfortunately, I cannot set the rules in sonar for NLog now (need to move the repo to a new org)